### DISCUSSION TITLE
Where is now the content directory?

### DISCUSSION THREAD

**Discussion by realtebo** [nuxt/nuxt#17630] [General]
From nuxt 2 doc: https://nuxtjs.org/docs/directory-structure/content/ 

Can we still use content directory as data source?

**Comment by atinux** [comment]
It's coming very soon for Nuxt 3 (talking about a week of two)

**Reply by Benbinbin** [reply]
How about Docus, I found an [extension](https://marketplace.visualstudio.com/items?itemName=NuxtLabs.docus) about it in VSCode, it seems interesting.

**Reply by ManasMadrecha** [reply]
That's great. 😊 Excited to update @jainism-portal's several projects.

**Reply by Benbinbin** [reply]
now the [content](https://github.com/nuxt/content) v2 support the nuxt 3, and I wonder it's the exactly version of [content-next](https://v3.nuxtjs.org/community/roadmap#%EF%B8%8F-roadmap) showed in roadmap?

**Reply by onlime** [reply]
Great! Let's give them some more time to update the docs on https://content.nuxtjs.org - really hoping content v2 will deliver a similar feature set like v1. @Atinux any update on this?
